### DISCUSSION TITLE
Is there a way to order videos in History from most recent to oldest?

### DISCUSSION THREAD

**Discussion by Nono-Song** [FreeTubeApp/FreeTube#5691] [Q&A] [answered]
As title. Currently, it is ordered starting from the oldest video…

**Comment by absidue** [comment, marked as answer]
I presume you just imported your history, if that's the case it's a known issue that the history entries are displayed in the wrong order in the app right after the import, restarting FreeTube fixes it. The issue itself will will be fixed by: https://github.com/FreeTubeApp/FreeTube/pull/5666
